readme.txt

```diff
@@ -0,0 +1,15 @@
+=== WooCommerce Admin ===
+Contributors: automattic
+Tags: ecommerce, e-commerce, store, sales, reports, analytics, dashboard, activity, notices, insights, stats, woo commerce, woocommerce
+Requires at least: 5.0.0
+Tested up to: 5.0.3
+Stable tag: 1.0.0
+License: GPLv2
+License URI: https://github.com/woocommerce/wc-admin/blob/master/LICENSE.md
+
+A feature plugin for a modern, javascript-driven WooCommerce admin experience.
```

Comment:
This seems good for now, but should we create a follow-up issue to provide a better description (once we come up with it 😅 )?

Seems like this might be one of the driving factors for installs - https://make.wordpress.org/plugins/2018/03/06/understanding-readme-txt/

Reply:
Thanks for pointing this out. I definitely consider the readme temporary pending the discussions at p8eNEx-3Dl-p2.

I've created an issue at #1536.